library/common-library/templates/_hostnetwork.tpl

```diff
@@ -0,0 +1,34 @@
+{{- /*
+Abstraction of the hostNetwork toggle.
+This helper allows to override the global `.global.hostNetwork` with the value of `.hostNetwork`.
+Returns "true" if `hostNetwork` is enabled, otherwise fallbacks to a overridable function
+*/ -}}
+{{- define "common.hostNetwork" -}}
+{{- /* This allows us to use `$global` as an empty dict directly in case `Values.global` does not exists */ -}}
+{{- $global := index .Values "global" | default dict -}}
+
+{{- /* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */ -}}
+{{- if get .Values "hostNetwork" | kindIs "bool" -}}
+    {{- if .Values.hostNetwork -}}
+        true
+    {{- else -}}
+        false
+    {{- end -}}
+{{- else if get $global "hostNetwork" | kindIs "bool" -}}
+    {{- if $global.hostNetwork -}}
+        true
+    {{- else -}}
+        false
+    {{- end -}}
```

Comment:
Ditto

Reply:
Solved in [#76799db](https://github.com/newrelic/helm-charts/pull/750/commits/76799db5ccad9eef227078fea726a82d6d12ea35)